grails-app/controllers/org/pih/warehouse/core/DocumentController.groovy

```diff
@@ -578,8 +594,8 @@ class DocumentController {
         Map model = [document: document, inventoryItem: inventoryItem, location: location]
         String body = templateService.renderTemplate(document, model)
 
-        response.contentType = 'image/png'    
-        // TODO Move labelary URL to application.yml 
+        response.contentType = 'image/png'
```

Comment:
can't it be combined somehow in the response builder below?
I mean here:
```
        response.outputStream << Request.Post('http://api.labelary.com/v1/printers/8dpmm/labels/4x6/0/')
            .bodyString(body, ContentType.APPLICATION_FORM_URLENCODED)
            .execute()
```

Reply:
I have not changed this part of the code. This is just IntelliJ that did some formatting changes (trailing spaces or something like that)